apps/rig_kafka/lib/rig_kafka/config.ex

```diff
@@ -51,7 +51,7 @@ defmodule RigKafka.Config do
       consumer_topics: Map.get(config, :consumer_topics, []),
       server_id: Map.get(config, :server_id) || String.to_atom("rig_kafka_#{uuid}"),
       client_id: Map.get(config, :client_id) || String.to_atom("brod_client_#{uuid}"),
-      group_id: Map.get(config, :group_id) || "brod_group_#{uuid}",
+      group_id: Map.get(config, :group_id) || "brod_group_rig",
```

Comment:
Why is the default here different from the default in `config.exs`? Actually, I'm not sure a default makes sense here in the first place - after all, we already define defaults in the config files.

Reply:
Left it there due to tests, I'll remove it and put in test config.